pkg/gadgets/sigsnoop/tracer/tracer.go

```diff
@@ -0,0 +1,279 @@
+//go:build linux
+// +build linux
```

Comment:
I think we should follow the same structure by defining this as the `core` tracer as we said in https://github.com/kinvolk/inspektor-gadget/pull/506#discussion_r809523589. WDYT?

cc @mauriciovasquezbernal

Reply:
What do you mean by this?

Reply:
I think Jose means to define this package under the core/, so in case we want to implement a standard/ tracer we can do it without having to move code around in the future.

Reply:
If that so, I am not inclined to do it.
I totally understand we add a `standard` tracer when we convert existing gadget.
But for new one, we should focus on CO-RE only.
CO-RE is the present, standard is already the past and I do not think I will be proven wrong if I say one day we will have remove all traces of standard tracer from our code base.

Reply:
There is a dedicated folder for the core tracer even if the standard one does not exist yet:

```bash
$ tree pkg/gadgets/fsslower/
pkg/gadgets/fsslower/
├── gadget.go
├── tracer
│   ├── core
│   │   ├── bpf
│   │   │   ├── fsslower.bpf.c
│   │   │   ├── fsslower.h
│   │   │   └── Makefile
│   │   ├── fsslower_bpfel.go
│   │   ├── fsslower_bpfel.o
│   │   └── tracer.go
│   └── interface.go
└── types
    └── types.go

4 directories, 9 files
```

Reply:
My motivation is not that we are going to add the standard version (maybe we will not) but having the same directory structure for all the gadgets. When people will go through the code, they will immediately think "_Why it is structured in a different way? Is there something special with it?_" ... I think it is important to have consistency and uniformity across the project.

Reply:
I will add a directory because this change already exists for `fsslower`.
But I do not really see the advantage here.